dts/riscv32/microsemi-miv.dtsi

```diff
@@ -44,14 +44,15 @@
 
 		plic: interrupt-controller@40000000 {
 			#interrupt-cells = <1>;
-			compatible = "riscv,plic0";
+			compatible = "sifive,plic-1.0.0";
```

Comment:
Just checking, its the case that microsemi-miv is using SiFive PLIC?

Reply:
Yes, that's correct